pkg/provider/azure_storageaccount.go

```diff
@@ -72,6 +72,9 @@ type AccountOptions struct {
 	SubnetName                              string
 	AccessTier                              string
 	MatchTags                               bool
+	EnableBlobVersioning                    *bool
```

Comment:
since we already have `SoftDeleteBlobs`, `SoftDeleteContainers`, do we need this `EnableBlobVersioning` field?

Reply:
I think so. Blob Versioning is similar to snapshot which makes customer able to rollback to a specific version.